### PR TITLE
Fix past deliveries accordions

### PR DESCRIPTION
**Pull request by immora**
Before: 
initial_bug

After:
fixed_bug

### CODE REVIEW

**Comment by mcmpp** [comment]
Hi @immora , thanks so much for your contribution. 
There is only one missing thing. For the version, we have the bumpversion.sh script that upgrade the version in all files that is needed. Please, if you run it and push the result I accept this pull request.

**Comment by immora** [comment]
> Hi @immora , thanks so much for your contribution.
> There is only one missing thing. For the version, we have the bumpversion.sh script that upgrade the version in all files that is needed. Please, if you run it and push the result I accept this pull request.

Hello @mcmpp! Done :) Although I'm not sure if the version is totally correct, as there exists tag with 2.4.0.... But I used the bash script like you wrote.

LG,
Kasia

**Comment by mcmpp** [comment]
@immora , thanks so much. I will approve and close. As you already know the project and use it, do not hesitate to report other bugs, fix them, helping out with translations or any other task that you could think is helpful. 
A good way to start, if you haven't already read it is https://github.com/OpenOlitor/OpenOlitor/blob/prod/CONTRIBUTING.md

Have a great evening

**Review by mcmpp — APPROVED** [review]

File: variables.json [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {   
     "ENVIRONMENTS" : {
         "dev": {
-            "m1": "http://localhost:9003/m1/",
-            "m2": "http://localhost:9003/m2/"
+            "m1": "http://localhost:8080/api-csa1/"
```
Comment (mcmpp) [inline variables.json]: Sorry, I have accepted it before seeing this issue. This file should not be modified

**Review by mcmpp — CHANGES_REQUESTED** [review]

File: variables.json (hunk 1 above)
Comment (immora) [reply variables.json]: Reverted and version bumped again after pulling new prod branch

**Review by mcmpp — APPROVED** [review]

**Pull request merged**